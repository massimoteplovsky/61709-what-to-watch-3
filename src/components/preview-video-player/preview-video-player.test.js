import React from 'react';
import renderer from 'react-test-renderer';

import PreviewVideoPlayer from './preview-video-player.jsx';

it(`<PreviewVideoPlayer /> is rendered correctly`, () => {

  const tree = renderer.create(
      <PreviewVideoPlayer>
        <video/>
      </PreviewVideoPlayer>, {
        createNodeMock: () => {
          return {};
        }}
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
