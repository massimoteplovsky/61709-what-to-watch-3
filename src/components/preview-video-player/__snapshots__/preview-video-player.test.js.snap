// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreviewVideoPlayer /> is rendered correctly 1`] = `<video />`;
